<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cfg_Page</class>
 <widget class="QWidget" name="cfg_Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_configOutform_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>模块选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cfgNextBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QComboBox" name="configOutformBox_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>glut</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>glfw</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>egl</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sdl</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>glx</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_configOutform_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>API 版本</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_configArch">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>硬件平台</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>配置选择</string>
   </property>
  </widget>
  <widget class="QComboBox" name="configOutformBox_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>coverage</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>function</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>error</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>limit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>shader</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_configOutform">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>输出格式</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cfgSelfCheck">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>是否自选用例</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="configDumpBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>dump输出</string>
   </property>
  </widget>
  <widget class="QComboBox" name="configOutformBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>All</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL1.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL1.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL1.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL2.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL2.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL2.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL3.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL3.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL3.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL3.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL4.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL4.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL4.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL4.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL4.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL4.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OGL4.6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GLES1.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GLES2.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GLES3.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GLES3.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GLES3.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GLES3.3</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>可选项</string>
   </property>
  </widget>
  <widget class="QComboBox" name="configOutformBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>html</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>excel</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_configLibpath">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>库路径</string>
   </property>
  </widget>
  <widget class="QComboBox" name="configArchBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>x86_64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mips64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>armv8</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_configOutform_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>窗口系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cfgExitBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="configGDBBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>开启GDB</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cfgAPICheck">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>按照API选择</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
